security/bug-bounty.md

```diff
@@ -21,7 +21,7 @@ Vulnerabilities are eligible for the bug bounty program if they meet a minimum s
 
 In general, the following would not meet the threshold for severity:
 
-* Denial of service, spamming, or causing annoyance.
+* Denial of service, spamming, email content modifications, or causing annoyance.
 * Issues with Clever's email server configuration (SPF, DKIM, and DMARC) or name server configuration (DNSSEC).
```

Comment:
Maybe we can shift the `email content modifications` to the email bullet point below?

```suggestion
* Denial of service or general actions with the sole intent of causing annoyance.
* Issues surrounding emails:
  * infrastructure-wise: Clever's email server configuration (SPF, DKIM, and DMARC)
  * product-wise: content modification
* Name server configuration (DNSSEC, CAA, etc.).
```